Work around an hlint bug where it does not recognise CPP when
Haskell2010 is passed to it; see

    https://github.com/ndmitchell/hlint/issues/1360

--- a/.github/workflows/haskell-ci.yml.orig
+++ b/.github/workflows/haskell-ci.yml
@@ -247,7 +247,7 @@
           $CABAL v2-test $ARG_COMPILER $ARG_TESTS $ARG_BENCH all --test-show-details=direct
       - name: hlint
         run: |
-          if [ $((HCNUMVER >= 90000 && HCNUMVER < 90200)) -ne 0 ] ; then (cd ${PKGDIR_xmonad_contrib} && hlint -h ${GITHUB_WORKSPACE}/source/.hlint.yaml -XHaskell2010 .) ; fi
+          if [ $((HCNUMVER >= 90000 && HCNUMVER < 90200)) -ne 0 ] ; then (cd ${PKGDIR_xmonad_contrib} && hlint -h ${GITHUB_WORKSPACE}/source/.hlint.yaml -XHaskell2010 -XCPP .) ; fi
       - name: cabal check
         run: |
           cd ${PKGDIR_xmonad_contrib} || false
